distance_calculation.js

```diff
@@ -0,0 +1,34 @@
+const methods = {
+
+      calculateGreatCircleDistance : 
+      function (latitude1, longitude1, latitude2, longitude2) {
```

Comment:
Usually varName1, varName2,... are not great names for variables. Can you try to replace them with something else, please?